dotcom-rendering/src/client/userFeatures/userBenefitsApi.ts

```diff
@@ -0,0 +1,41 @@
+import { isObject } from '@guardian/libs';
+import {
+	getOptionsHeadersWithOkta,
+	type SignedInWithCookies,
+	type SignedInWithOkta,
+} from '../../lib/identity';
+import { fetchJson } from './fetchJson';
+import type { UserBenefits } from './user-features';
+
+type UserBenefitsResponse = {
+	benefits: string[];
+};
+export const syncDataFromUserBenefitsApi = async (
+	signedInAuthStatus: SignedInWithOkta | SignedInWithCookies,
+): Promise<UserBenefits> => {
+	const response = await fetchJson(
+		`${
+			window.guardian.config.page.userBenefitsApiUrl ??
+			'https://user-benefits.code.dev-guardianapis.com/benefits/me'
+		}/me`,
```

Comment:
will that not end up with `/me/me` at the end of the url?

Reply:
are we likely to have multiple services used on that API or is it worth just making the url one fixed string?